blotoutSDK/src/main/java/com/analytics/blotout/geasture/Gesture.java

```diff
@@ -271,7 +260,7 @@ public void setLongPressTimeout(long longPressTimeout) {
 
   private void checkNotNull(Object object, String message) {
     if (object == null) {
-      throw new IllegalArgumentException(message);
+      //throw new IllegalArgumentException(message);

```

Comment:
what should we do here? log?

Reply:
made changes to implemented  it in another way. 

-> There is no chances to receive listener object as null.